### PR TITLE
Skipping

### PR DESCRIPTION
**Pull request by NicGiles**
added skip function

### CODE REVIEW

**Review by alakijaayo — APPROVED** [review]

**Review by Aiman15049 — APPROVED** [review]
fantastic

File: server/app.js [hunk 1]
```diff
@@ -195,6 +195,9 @@ io.on('connection', function(socket) {
   socket.on('vote up', function(uri) {
     queue.vote(uri, 1);
   });
+  socket.on('skip', function(uri) {
+     queue.skip(uri);
```
Comment (jonurry) [inline server/app.js]: you can change this to queue.playNextTrack()

File: server/src/queue.js [hunk 2]
```diff
@@ -57,6 +58,16 @@ class Queue {
     this.playCurrentTrack();
   }
 
+  skip() {
```
Comment (jonurry) [inline server/src/queue.js]: you can remove this

File: server/src/queue.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ class Queue {
     this.playCurrentTrack = this.playCurrentTrack.bind(this);
     this.playNextTrack = this.playNextTrack.bind(this);
     this.sortNextByVotes = this.sortNextByVotes.bind(this);
+    this.skip = this.skip.bind(this);
```
Comment (jonurry) [inline server/src/queue.js]: you can remove this

**Review by jonurry — APPROVED** [review]
Nice!

**Pull request merged**